src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-is-hover: #00acdc;
+}
+
 body {
```

Comment:
I would probably provide more generous CSS styles reset like:
```
*, *::after, *::before {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```
but that's a good start 😄 

[more about custom CSS resets](https://www.joshwcomeau.com/css/custom-css-reset/)

Reply:
Is it better to use normalize css for that?

Reply:
For this kind of project I would stick to the basics, and those simple styles are more than enough, but for larger projects I would definitely take that into consideration 😉